.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: NodeJS with Grunt
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Build
+      run: |
+        npm install
+        grunt lint
+        grunt test
```

Comment:
Does this work?
[Your latest job](https://github.com/peternewman/openvegemap/runs/4695355279?check_suite_focus=true) says that it did not run any test:
```
Running "qunit:files" (qunit) task
>> 0 tests completed with 0 failed, 0 skipped, and 0 todo. 
>> 0 assertions (in 0ms), passed: 0, failed: 0
```

You probably need to run `grunt` first to generate the compiled test files.

Reply:
Forget what I said. If you run `yarn install`, it should call `webpack` and build the compiled JS automatically.

Reply:
Ah, I was a bit confused. Curiously some of the previous runs had run fine.